po/README.md

```diff
@@ -1,22 +1,214 @@
 # Translating Extension Manager
+Translations contributions is always welcome!
 
-## Translation Guide
-Work in progress - please check back later
+If you totally new to GTK software transitioning or software development in general, fear not! Everything here explained as detailed, as possible!
+
+If you already have experience with translating GTK applications, remember that Extension Manager is written in C, so you need to rebuild it every time, when you do changes in `po` files.
+
+## How to add new translation
+
+1. You need software to translate Extension Manager (in this case, we will use [POEdit](https://poedit.net/)) and [GNOME Builder](https://apps.gnome.org/Builder/) to build and test your translation.
+2. Clone Extension Manager repository to your device.
+
+* *Using git:*
+```bash
+git clone git@github.com:mjakeman/extension-manager.git
+```
+
+* *Using Github:*
+
+1. Click green button `Code`.
+2. In opened menu click `Download ZIP`.
+3. Save it in your `~/` home folder.
+4. Unpack archive with archive manager of your choice.
+5. Rename folder that will be unpacked from archive to `extension-manager`.
+
+* *Using GNOME Builder:*
+
+1. Run GNOME Builder.
+2. Click `Clone Repository` button on bottom.
+3. In `Repository URL` paste: `https://github.com/mjakeman/extension-manager`
+4. Set `Location` to `~/`
+
+Since later we will use GNOME Builder again, you can leave it opened.
+
+
+3. You need `pot` file, that lies inside `./po` folder. Make sure, that [pot file is up-to-date](#regenerate-pot-files), before proceeding.
+4. Open it with POEditor.
+5. Click on button on bottom of window `Create a new translation`, select your desired language.
+6. Press `Ctrl+S` to save new file, place it inside `./po` folder with filename that POEdit gave you.
+7. Open `./po/LINGUAS` file with any text editor, and add there locale code of your language. For example, if you want to add Ukrainian translation, POEdit will suggest you filename `uk.po`, so in `LINGUAS` file, you need to add `uk`. And, please, keep locale codes in alphabetical order.
+
+But sometimes, you might want to specify variant for your language. For example, for Russian in Russia Federation, you need to set locale to `ru_RU`, instead of just `ru`.
```

Comment:
```suggestion
But sometimes, you might want to specify a variant for your language. For example, for Russian in Russia Federation, you need to set locale to `ru_RU`, instead of just `ru`.
```